src/Finder/ModuleFileFinder.php

```diff
@@ -46,4 +57,45 @@ protected function _filter(SplFileInfo $fileInfo)
     {
         return $fileInfo->isReadable();
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function rewind()
+    {
+        $this->iterator = new ArrayIterator($this->_getPaths());
```

Comment:
The alternative is to create the iterator on construction, since there is no interface backing a file finder and hence no other public methods to "begin" iteration.

This method also allows the finder to cater for changes in the file system on rewind. This has been decided on a voice call to be the best approach for now, until/if caching of the result from `$this->_getPaths()` is implemented.